C-Sharp/VectorDistanceCalculation/PE11VectorDistanceCalculation/Menu.cs

```diff
@@ -10,23 +10,31 @@ class Menu
     {
         private static int selection;
         private static int[] validSelections = { 1, 2, 3 };
+        //Have you guys gone over unit tests?
+        //I've seen a few bugs in your code but unit tests will surface those bugs
         public static void Run()
         {
             displayWelcomeMessage();
             
             while (true)
             {
-                selection = getValidUserInput();
+                selection = getValidUserInput();//start all letters with capital letter
+                //rule of thumb but all switch statesments should have a a default
+                //even though your switch statement is ok now but what happens if future you or another dev adds another valid input?
```

Comment:
Got it! GetValidUserInput only returns a value in the ValidSelections array, so I figured I wouldn't need a default case, but it makes sense to add it, to prevent any unexpected error/behavior in the future.